cmd/gardener-controller-manager/app/gardener_controller_manager.go

```diff
@@ -127,22 +127,6 @@ func (o *Options) validate(args []string) error {
 	return nil
 }
 
-func (o *Options) applyDefaults(in *config.ControllerManagerConfiguration) (*config.ControllerManagerConfiguration, error) {
```

Comment:
Not related to this PR, but can you check line 84, though?
It seems wrong to add the gardener API types to the kubernetes `scheme.Scheme` 😄 
```go
if err := gardencorev1beta1.AddToScheme(scheme.Scheme); err != nil {
```